lldb/source/Plugins/SymbolLocator/DebugSymbols/SymbolLocatorDebugSymbols.cpp

```diff
@@ -776,6 +776,10 @@ std::optional<FileSpec> SymbolLocatorDebugSymbols::LocateExecutableSymbolFile(
       exec_fspec ? exec_fspec->GetFilename().AsCString("<NULL>") : "<NULL>",
       arch ? arch->GetArchitectureName() : "<NULL>", (const void *)uuid);
 
+  Progress progress(
+      "Locating external symbol file",
```

Comment:
Are we looking for an executable here, or the dSYM file? Should be title be "Locating executable file"?

Reply:
We're looking for the symbol file (although dsymForUUID can also get you the symbol rich binary). This is the same message already emitted by `SymbolLocatorDefault::LocateExecutableSymbolFile`.